CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # CHANGELOG
 
+# 1.12.0
+* Feature - Using interest rate on minimum instalment value [#490](https://github.com/ebanx/woocommerce-gateway-ebanx/pull/490)
+* Feature - Refactor EBANX query router [#487](https://github.com/ebanx/woocommerce-gateway-ebanx/pull/487)
+* Feature - Added a minimal value setting on settings [#477](https://github.com/ebanx/woocommerce-gateway-ebanx/pull/477)
+* Feature - Changed cookie to localStorage to save flags [#476](https://github.com/ebanx/woocommerce-gateway-ebanx/pull/476)
+* Feature - Removed doc caches [#492](https://github.com/ebanx/woocommerce-gateway-ebanx/pull/492)
```

Comment:
This doesn't get in as a feature XD
It's almost a revert of a bad binary commit.